src/components/views/right_panel/PinnedMessagesCard.tsx

```diff
@@ -110,6 +110,16 @@ const PinnedMessagesCard = ({ room, onClose }: IProps) => {
                 if (event && PinningUtils.isPinnable(event)) {
```

Comment:
Btw why are you checking for `event` here? L107 already asserts it to be present